### ISSUE TITLE
New CityIO version

### ISSUE THREAD

**Issue by crisjf** [CityScope/CS_Brix#36]
Todo:

* Ensure all trailing slashes are added for both POST and GET requests
* Add /Content-Type:application/json/ for POST requests
* New update endpoint: https://cityiotest.mirage.city/api/table/new_table
* Test `brix` using: https://cityiotest.mirage.city

Also note that POST requests append by default, so there is no need to post indicators or grid cells that have not changed.

**Comment by crisjf** [comment]
Old attributed `brix.Handler.append_on_post` needs to be dropped as well.

**Comment by crisjf** [comment]
@RELNO do you have an easy way to populate some of Yasushi's test [tables](https://cityiotest.mirage.city/api/tables/list/) in mirage.city with GEOGRIDs? I think I'm done with the core changes but I can't test the post requests until I have some GEOGRIDs to work with.

**Comment by RELNO** [comment]
not really, I guess `curl` any of the existing grids would do the trick

**Comment by crisjf** [comment]
@RELNO I really miss `dungeonmaster`

https://github.com/CityScope/CS_CityIO/issues/125

**Comment by crisjf** [comment]
I created another class to make testing easier. Documentation for this is still pending. 

The `User` class simulates a user updating the table by either i) flipping a random cell to a different type, ii) shuffling the types of the whole table, or iii) resetting the grid to its original value. It does all of this in a different thread to allow the module builder to continue working on their module.

Here's a short example
```
from brix.examples import Diversity
from brix import Handler, User

table_name = 'dungeonmaster'

U = User(table_name)
U.start_user()

H = Handler(table_name,quietly=False)
div = Diversity()
H.add_indicator(div)
H.listen()
```

Note that `U.start_user()` creates a new thread. To see the status, use `U.user_status()`. To stop the user, use `U.stop_user()`

**Comment by crisjf** [comment]
I wrote a unittest for the endpoints of the new api.

See here:
https://github.com/CityScope/CS_Brix/blob/cityio_update/test/test_api.py

**Comment by crisjf** [comment]
This seems to be working now. 

There was a small issue that indicators without a name were not working when added to a Handler, but that was fixed by ca9176fe02de6bd7a71fc98b9237212ca17fcebb but is not yet updated in pip.
